modules/openapi-generator/src/main/resources/python-flask/tox.mustache

```diff
@@ -6,4 +6,4 @@ deps=-r{toxinidir}/requirements.txt
      -r{toxinidir}/test-requirements.txt
 
 commands=
-   {{^useNose}}pytest --cov={{{packageName}}}{{/useNose}}{{#useNose}}nosetests{{/useNose}}
\ No newline at end of file
+   {{^useNose}}pytest --cov={{#srcLayout}}src/{{/srcLayout}}{{{packageName}}}{{/useNose}}{{#useNose}}nosetests{{/useNose}}
```

Comment:
Did you test this with useNose=True also?
You might need to hard code in a path for nosetests now that the code is deeper.

Reply:
hmm. I don't know nose.
But from a quick google, it looks like it will use lib and src directories by default.

Reply:
If you can point me to an option for nosetests to set the python sources root, I'm happy to add that.
From what I can tell, you have to add it inside the test code: https://stackoverflow.com/a/30501263
But there isn't any test code to work with. It looks like all of the tests are written with pytest in mind (from my limited understanding of nose).
The things that take `useNose` into account are `tox.mustache` and `test-requirements.mustache`. It looks like the flask template for travis expects to run tests with nosetests without regards to `useNose`.

De-tangling the nose stuff would probably be good for a separate PR (by someone who actually uses / understands nose).